community/resources/zero-trust-whitepaper/v1/cloud-native-zero-trust-whitepaper.md

```diff
@@ -0,0 +1,921 @@
+4. [Techniques and Best Practices](#4-techniques-and-best-practices)
+   - [Protect Data Confidentiality and Integrity](#protect-data-confidentiality-and-integrity)
+   - [Prevent Unauthorized System Use](#prevent-unauthorized-system-use)
+   - [Establish Limited Trust](#establish-limited-trust)
+
+- [Conclusion](#conclusion)
+- [Appendix](#appendix)
+  - [Glossary](#glossary)
+  - [References and Citations](#references-and-citations)
+  - [Contributors](#contributors)
+  - [Reviewers](#reviewers)
+  - [Acknowledgments](#acknowledgments)
+  - [Helpful Links](#helpful-links)
+
+**Note**: All tools, projects, libraries mentioned in the paper are mentioned as examples when describing a relevant section. Hence, such mention **should not be** taken as an official recommendation by TAG Security or CNCF. Readers should evaluate adoption of such tools, projects or libraries based on their own understanding and threat model of the system under review.
+
+# Abstract
+
+Contrary to what the name might suggest, the real world application of “Zero Trust” is far more nuanced than simply *trusting nothing*. The Zero Trust defense strategy assumes that the internal network is not to be trusted. This contrasts with a perimeter-based defense, which is designed to construct a trustworthy internal network. Instead, we can introduce measures to evaluate trustworthiness, then use such evaluations to control the network communications and its connected devices.
+
+While many of the well-worn concepts behind Zero Trust apply to *any* system, there remains a gap with regards to discussing Zero Trust from a Cloud Native perspective. This document seeks to codify the philosophy alongside an ideal design for implementing it in a Cloud Native system.
+
+The authors have compiled their experience and research findings into a set of principles and approaches. While many of the concepts herein are a distillation of past publications, extending those findings has led to a new proposal to standardize the generation and utilization of “Confidence Levels” as a data type. Confidence Levels quantify the trustworthiness of entities within a system, allowing for more dynamic and responsive security measures.
+
+Confidence Levels can be produced by “Active Observers,” a previously unnamed category of tools. Active Observers continuously monitor and analyze the security-related attributes and behaviors of entities in the system to quantify trustworthiness.
+
+By applying the philosophies of Zero Trust to an entire Cloud Native system with consideration paid to the unique context, this paper crafts a Cloud Native Zero Trust Architecture design. Using the latest technologies, it is becoming possible to build a system in such a way that Active Observers assign Confidence Levels to every entity in the system. This enables the architecture to adapt in real-time to emerging threats and anomalies, reinforcing the Zero Trust stance.
```

Comment:
Lets replace to  singular form philosophy. 

As for Active Observers, IMHO this is becoming clearer later in the doc, we name what Active Observers are needed and how they are used. I think it is fine to keep this "as is" and I actually like the fact that it may intrigue a reader to read further about it in the doc...  :)